site/src/components/WorkspaceSchedule/WorkspaceSchedule.tsx

```diff
@@ -15,27 +17,31 @@ import { extractTimezone, stripTimezone } from "../../util/schedule"
 import { isWorkspaceOn } from "../../util/workspace"
 import { Stack } from "../Stack/Stack"
 
+dayjs.extend(advancedFormat)
 dayjs.extend(utc)
 dayjs.extend(duration)
 dayjs.extend(relativeTime)
+dayjs.extend(timezone)
 
 export const Language = {
   autoStartDisplay: (schedule: string | undefined): string => {
     if (schedule) {
       return cronstrue.toString(stripTimezone(schedule), { throwExceptionOnParseError: false })
+    } else {
+      return "Manual"
     }
-    return "Manual"
```

Comment:
**Review**:

Just meeting the FE code convention of using explicit `else` here.